docs/content/administration/config-cheat-sheet.en-us.md

```diff
@@ -892,6 +892,21 @@ Default templates for project boards:
 - `PROTOCOL`: **tcp**: Set the protocol, either "tcp", "unix" or "udp".
 - `ADDR`: **:7020**: Sets the address to connect to.
 
+## Audit Log (`audit`)
+
+- `ENABLED`: **false**: Enable logging of audit events
+- `APPENDER`: **\<empty\>**: Comma seperated list of appenders to log to (possible values: `log` or `file`).
```

Comment:
What does the `log` value do? Log to default loggour output (`console`).

Reply:
Yes, see the docs:

> The audit log supports different appenders:
>
> - `log`: Log events as information to the configured Gitea logging